ui/pages/onboarding-flow/create-password/create-password.js

```diff
@@ -85,13 +85,14 @@ export default function CreatePassword({
 
   const handlePasswordChange = (passwordInput) => {
     let confirmError = '';
+    let passwordInputError = '';
     const passwordEvaluation = zxcvbn(passwordInput);
     const passwordStrengthLabel = getPasswordStrengthLabel(
       passwordEvaluation.score,
       t,
     );
-    const passwordStrengthDescription = passwordStrengthLabel.description;
-    const passwordStrengthInput = t('passwordStrength', [
+    let passwordStrengthDescription = passwordStrengthLabel.description;
```

Comment:
These could still be `const`s with some ternaries, like
`const passwordStrengtDescription = CONDITION ? X : Y`.

As it is right now the logic and intention is not trivial to follow.